core/sts-3.3.md

```diff
@@ -290,6 +290,10 @@ an administrator has regained control of their server. This can be mitigated in
 by allowing for STS policy rejection as described in the "Client implementation
 considerations" section.
 
+* A 3rd party host that points to an STS-enabled host via DNS alias, where the 3rd party
+isn't listed on the server's certificate. This configuration would fail certificate validation
+even without STS, but users may be relying on it for unsecure access.
```

Comment:
The spec is using the "insecure" and "non-secure" nomenclature for plaintext connections, I suggest we pick one and stick to it instead of adding unsecure as a third one.